src/SchedulerSubsystem/Scheduler.java

```diff
@@ -43,7 +85,7 @@ private void processElevatorEvent(ElevatorStateEvent event) {
         if (isElevatorStopping(event)) // Stop elevator for a load
             new Thread(new Loader(event, transmitterToFloor, transmitterToElevator)).start();
         else // Keep moving
-            transmitterToElevator.send(new MoveElevatorCommand(event.elevatorNum()));
+            transmitterToElevator.send(new MoveElevatorCommand(event.elevatorNum(), getElevatorDirection(event)));
```

Comment:
Null case: Elevator has no passengers, serves highest priority floor request.
Highest priority floor request -> could look for oldest floor request? (This requires quite a few changes, leave as is for this PR but it's a blocking feature for this iteration)